airflow/configuration.py

```diff
@@ -276,6 +274,8 @@ def has_option(self, section, option):
             # UNSET to avoid logging a warning about missing values
             self.get(section, option, fallback=_UNSET)
             return True
+        except NoSectionError:
+            return False
```

Comment:
Do you need to add this change here? If you do please add `[AIRFLOW-4831]` (`[AIRFLOW-3991][AIRFLOW-4831] ...`) in to your commit message and add a test for the changed/improved `has_option` behaviour too.

Reply:
I'm going to merge #5455 just as soon as the tests are geen so this change will already be made.